src/unstable-temp/DownloadImageModal/DownloadImageModal.tsx

```diff
@@ -111,6 +112,7 @@ export interface UnstableTempDownloadImageModalProps {
 		}
 	>;
 	authToken?: string;
+	docsIcon: IconProp;
```

Comment:
```suggestion
	docsIcon?: IconProp;
```
I would not make it `required` otherwise it may no longer work in some projects. Let's set it optional and if it is not passed we use the free one.